ckanext/data_qld/controller.py

```diff
@@ -85,7 +85,7 @@ class DataQldDataset(PackageController):
     def _get_context(self):
         return {'model': model, 'session': model.Session, 'user': c.user, 'for_view': True, 'auth_user_obj': c.userobj}
 
-    def _is_dataset_private(self, id):
+    def _is_dataset_public(self, id):
```

Comment:
It's more about "visible" than "public" (ie either it's public, or we're logged in with sufficient privileges), but ok.